### PR TITLE
data: akshatshukla13

### PR DESCRIPTION
**Pull request by akshatshukla13**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5403"></a>

### CODE REVIEW

File: akshatshukla13.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Akshat Shukla",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "As An enthusiastic tech aficionado, I am constantly fueled by my passion to explore and discover",
+  "avatar": "https://media.licdn.com/dms/image/D5603AQFA-E-9DFGSUA/profile-displayphoto-shrink_800_800/0/1668493033974?e=1684368000&v=beta&t=RRQZUtcrRu3hXT6yIZoGZ65ydkLUFVakMK4C43lWbPE",
```
Comment (Sindhuinti) [inline akshatshukla13.json:6]: No need to add your avatar here. It will be auto imported from github.

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: akshatshukla13.json (hunk 1 above)
Comment (kumarsonsoff3) [inline akshatshukla13.json:6]: As of our recent update, we no longer need the `displayStatsPublic` & `avatar` keys. So please remove these fields from your json. 

```suggestion
  "type": "personal",
  "bio": "As An enthusiastic tech aficionado, I am constantly fueled by my passion to explore and discover",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Also add your file to data directory. To do so, you need to [rename your file here](https://github.com/akshatshukla13/LinkFree/edit/akshatshukla13-patch-1/akshatshukla13.json) to `data/akshatshukla13.json`. 

Change `akshatshukla13.json`  - >  `data/akshatshukla13.json`

File: data/akshatshukla13.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Akshat Shukla",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "As An enthusiastic tech aficionado, I am constantly fueled by my passion to explore and discover",
+  "avatar": "https://media.licdn.com/dms/image/D5603AQFA-E-9DFGSUA/profile-displayphoto-shrink_800_800/0/1668493033974?e=1684368000&v=beta&t=RRQZUtcrRu3hXT6yIZoGZ65ydkLUFVakMK4C43lWbPE",
```
Comment (SaraJaoude) [inline data/akshatshukla13.json:6]:
```suggestion
```

File: data/akshatshukla13.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Akshat Shukla",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/akshatshukla13.json:3]:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**